images/rootfs-acrn.yml.in

```diff
@@ -3,8 +3,8 @@ kernel:
   cmdline: "rootdelay=3"
 init:
   - linuxkit/init:v0.5
-  - linuxkit/runc:v0.5
-  - linuxkit/containerd:v0.5
+  - linuxkit/runc:69b4a35eaa22eba4990ee52cccc8f48f6c08ed03
+  - linuxkit/containerd:09553963ed9da626c25cf8acdf6d62ec37645412
```

Comment:
Can we use the latest release of linuxkit here or do we have to latch onto this?

Reply:
I tried with the latest version on linuxkit (v0.7) and faced the same issue. But with the latest tags as per https://hub.docker.com/r/linuxkit/runc/tags and https://hub.docker.com/r/linuxkit/containerd/tags snapshot issue was resolved.

Reply:
Oh, ok